fair/cli.py

```diff
@@ -324,14 +324,16 @@ def install(debug: bool, force: bool, directory: str, version: str):
 @registry.command()
 @click.option("--debug/--no-debug", help="Run in debug mode", default=False)
 @click.option("--port", help="port on which to run registry", default=8000)
-def start(debug: bool, port: int) -> None:
+@click.option("--address", help="Address on which to run registry", default="127.0.0.1")
```

Comment:
Win 10, Py 10

Works with `--address 127.0.0.1` not with `192.168.1.101` my laptop address at the time - gives `Bad Request (400)` so serving something! The bit, the click CLI is clearly OK.

Reply:
This is an issue with allowed hosts: [base_settings.py](https://github.com/FAIRDataPipeline/data-registry/blob/main/drams/base_settings.py#L12)

The address will need adding to the allowed hosts or allowing all hosts.
ALLOWED_HOSTS = ['*']
ALLOWED_HOSTS = ['127.0.0.1', 'localhost', '192.168.0.101']

This can be added to [local-settings.py](https://github.com/FAIRDataPipeline/data-registry/blob/main/drams/local-settings.py)

Or we could import local settings into a new settings file and put it there.

Open to suggestions on where and how we add this in?

Reply:
My intuition here is that this feature is going to be used infrequently or by expert users for either running in containers or deploying a "remote" registry. @bruvio's suggestion looks good (with my limited Django). Thanks! I agree that allowing all hosts by default is unlikely to be good practise. 

I think if you go with the suggestion, it should be documented somewhere. I guess the other way might be to just document what manual changes are needed to these files to allow the host specified by the CLI.

Reply:
> Hi sorry if I jump in.
> 
> I would not allow any hosts.
> 
> I would edit the Django settings to be like
> 
> ALLOWED_HOSTS = [] ALLOWED_HOSTS.extend( filter(None, os.environ.get("ALLOWED_HOSTS", "").split(",")) )
> 
> And then pass allowed host as env variable.
> 
> This would allow to have less maintenance overhead in the Django app for future changes.
> 
> You could also create a sys-env variable (prod/testing/staging) and have different Django settings.

Thanks @bruvio 

Agree this is a good solution, I would probably set allowed hosts in the settings to ['127.0.0.1', 'localhost'] prior to extending it with the environment variable so if the environmental variable is not set, it will still run on both.

I'll put in a PR on the registry and we can add the address to the environment variable in the start registry function